internal/controller/postgrescluster/helpers_test.go

```diff
@@ -67,11 +65,7 @@ func init() {
 
 // marshalMatches converts actual to YAML and compares that to expected.
 func marshalMatches(actual interface{}, expected string) cmp.Comparison {
```

Comment:
This one is used over 40 times. I wanted to avoid the conflicts that would come from such a big change. From the commit message:

> Replacing every call to "marshalMatches" is very likely to conflict with other work, so leave those names in place for now.

```
● ag -c marshalMatches -- internal/controller/postgrescluster
internal/controller/postgrescluster/volumes_test.go:3
internal/controller/postgrescluster/cluster_test.go:10
internal/controller/postgrescluster/topology_test.go:1
internal/controller/postgrescluster/pgbackrest_test.go:2
internal/controller/postgrescluster/pgbouncer_test.go:6
internal/controller/postgrescluster/postgres_test.go:2
internal/controller/postgrescluster/helpers_test.go:2
internal/controller/postgrescluster/instance_test.go:8
internal/controller/postgrescluster/pgadmin_test.go:9
internal/controller/postgrescluster/patroni_test.go:3
```